nix/pkgs/images/services/default.nix

```diff
@@ -0,0 +1,109 @@
+# It would be cool to produce OCI images instead of docker images to
+# avoid dependency on docker tool chain. Though the maturity of OCI
+# builder in nixpkgs is questionable which is why we postpone this step.
+#
+# We limit max number of image layers to 42 because there is a bug in
+# containerd triggered when there are too many layers:
+# https://github.com/containerd/containerd/issues/4684
+
+{ stdenv
+, busybox
+, dockerTools
+, e2fsprogs
+, git
+, lib
+, xfsprogs
+, utillinux
+, mayastor-release
+, mayastor-debug
+}:
+let
+  versionDrv = import ../../../lib/version.nix { inherit lib stdenv git; };
+  version = builtins.readFile "${versionDrv}";
+  env = stdenv.lib.makeBinPath [ busybox xfsprogs e2fsprogs utillinux ];
+
+  # common props for all service images
+  serviceImageProps = {
+    tag = version;
+    created = "now";
+    config = {
+      Env = [ "PATH=${env}" ];
+    };
+  };
+in
+{
+  # Release image of kiiss service.
```

Comment:
Similar question about the `-image` postfix, will we see a non `image` variant?

Reply:
Again, I think maybe I should just put all the images into the same place and drop the postfix.

Reply:
Done - I have removed the `-image` postfix now.